src/CommandingOfficers/Commander.java

```diff
@@ -365,7 +365,10 @@ public int getIncomePerTurn()
     ArrayList<UnitModifier> output = new ArrayList<UnitModifier>();
     output.add(this);
     output.addAll(unitMods);
-    output.addAll(this.army.getModifiers());
+    if( null != army )
+      output.addAll(army.getModifiers());
+    else
+      System.out.println("WARNING!: Getting Commander modifiers when CO.army == null!");
```

Comment:
a) Commanders don't have modifiers anymore?
2) Did this branch really need to go on top of the army branch?
z) Is this really a warning case? Is there always (supposed to be) an Army in the picture?

Reply:
...nice list labels.
a) It's Commander *`UnitModifiers`*. I guess I could specify that.
2) Yes, because Army added the multi-CO UI (because that's kinda critical for using `Army`)
z) One of the AIs hit that during game init once, which is fine(?) because there won't be any modifiers in `Army` at that point other than the environment hooks.

Reply:
If a Commander will always have an army, it seems strange that an AI would be able to act before an army was available. Might be worth an issue to investigate later?

Reply:
Also I just tried running and it fails to launch due to a missing army now:
```java
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException: Cannot read field "cos" because "this.myGame.activeArmy" is null
	at UI.Art.SpriteArtist.SpriteMapView.<init>(SpriteMapView.java:96)
	at UI.Art.SpriteArtist.SpriteEngine.createMapView(SpriteEngine.java:31)
	at UI.PlayerSetupController.handlePlayerSetupInput(PlayerSetupController.java:134)
	at UI.PlayerSetupController.handleInput(PlayerSetupController.java:86)
	at UI.GameOptionSetupController.handleInput(GameOptionSetupController.java:52)
	at UI.MapSelectController.handleInput(MapSelectController.java:37)
	at UI.MainUIController.handleInput(MainUIController.java:65)
	at Engine.Driver.keyPressed(Driver.java:115)
```